features/networking/multus.feature

```diff
@@ -991,3 +991,36 @@ Feature: Multus-CNI related scenarios
     When I execute on the pod:
       | /usr/sbin/ip | a |
     Then the output should contain "192.168.1"
+
+  # @author weliang@redhat.com
+  # @case_id OCP-25910
+  @admin
+  Scenario: Assign static MAC address using pod annotation				 
+    # Make sure that the multus is enabled
+    Given the multus is enabled on the cluster
+
+    # Create the net-attach-def via cluster admin
+    Given I have a project
+    When I run the :create admin command with:
+      | f | https://raw.githubusercontent.com/weliang1/Openshift_Networking/master/Features/multus/runtimeconfig-def-mac.yaml |
```

Comment:
Personal repository is used

Reply:
Wait for https://github.com/openshift-qe/v3-testfiles/pull/1206